src/components/cards/JufoExpertCard.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useContext } from 'react';
+import CardBase from '../base/CardBase';
+import { Text, Title } from '../Typography';
+import context from '../../context';
+import classes from './JufoExpertCard.module.scss';
+import Button from '../button';
+import { ExpertOverviewModal } from '../Modals/ExpertOverviewModal';
+import EditExpertProfileModal from '../Modals/EditExpertProfileModal';
+
+export const JufoExpertCard: React.FC = () => {
+  const modalContext = useContext(context.Modal);
+  const userContext = useContext(context.User);
+
+  return (
+    <>
+      <CardBase highlightColor="#4E6AE6" className={classes.baseContainer}>
+        <div className={classes.container}>
+          <div className={classes.matchInfoContainer}>
+            <Title size="h4" bold>
+              Willst du Jufo Experte*in werden?
```

Comment:
It would probably also here make sense here if we had a distinction between the case that the user is no expert yet and that he has already submitted some data.

Reply:
fixed with 685a1f5. But we still need some filler texts for "Denied" and "Pending" status as well as an Introduction text.